common/app/templates/headerInlineJS/bootCurl.scala.js

```diff
@@ -42,6 +42,7 @@ var curl = {
 @JavaScript(Static.js.curl)
 
 require([
+    'common/utils/config',
```

Comment:
Now the Ophan loader is dependent on utils/config, then the var should be guaranteed set for Ophan?